src/batch/src/executor/row_seq_scan.rs

```diff
@@ -303,18 +304,66 @@ impl<S: StateStore> RowSeqScanExecutor<S> {
             None
         };
 
-        // Scan all ranges concurrently.
-        let select_all = select_all(scan_ranges.into_iter().map(|scan_range| {
+        let (point_gets, range_scans): (Vec<ScanRange>, Vec<ScanRange>) = scan_ranges
```

Comment:
Is it possible to have both point get and range scan at the same time? 🤔 If not, there's no need to partition.

Reply:
It could be. For example: `select * from t where id = 1 or id > 10`